cdap-docs/examples-manual/source/examples/data-cleansing.rst

```diff
@@ -0,0 +1,170 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform DataCleansing Application
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _examples-word-count:
+
+==============
+Data Cleansing
+==============
+
+A Cask Data Application Platform (CDAP) example demonstrating incrementally consuming partitions of a partitioned file
+set using MapReduce.
+
+Overview
+========
+
+This application has a MapReduce which processes records from a partitioned file set into another partitioned file set,
+while filtering records that do not match a particular schema.
+
+- The ``DataCleansingService`` writes and serves data from the partitioned file sets.
+- The ``DataCleansingMapReduce`` processes the records, while filtering 'unclean' or invalid records.
+
+Let's look at some of these components, and then run the application and see the results.
+
+The Data Cleansing Application
+------------------------------
+
+As in the other :ref:`examples <examples-index>`, the components
+of the application are tied together by the class ``DataCleansing``:
+
+.. literalinclude:: /../../../cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java
```

Comment:
Sure. Add the line to the build.sh, with any hash in it. Run the docs build.sh command. You can either do it at the top level (cdap-docs/build.sh docs) or from within a manual's directory (examples-manual/build.sh docs).

In the output there will be an error message; it will have the correct hash. Copy and paste...